packages/components/src/tooltip/index.js

```diff
@@ -90,6 +95,7 @@ class Tooltip extends Component {
 
 	render() {
 		const { children, position, text, shortcut } = this.props;
+		const { isOver } = this.state;
```

Comment:
Why was this changed? Technically it's a violation of [the `no-unused-vars-before-return` rule](https://github.com/WordPress/gutenberg/blob/master/packages/eslint-plugin/docs/rules/no-unused-vars-before-return.md), but there is an explicit [exemption for property destructuring](https://github.com/WordPress/gutenberg/blob/d45c8ea9f3adafba76676a29f07f28dfd0f2d444/packages/eslint-plugin/rules/no-unused-vars-before-return.js#L39-L40) since they're more difficult (but foreseeably possible) to handle. It's for this temporary exemption only that this doesn't emit an ESLint error. In fact: Looking at this usage, I'm tempted to remove the exemption when the destructuring only contains a single property.

I might suggest it be put back where it was.

Reply:
Moved this back.
It was changed to make destructuring first in order to ease readability.
This is a frequent pattern I've seen through various react apps/libs.